src/sentry/tasks/app_store_connect.py

```diff
@@ -99,6 +99,7 @@ def inner_dsym_download(project_id: int, config_id: str) -> None:
             else:
                 create_difs_from_dsyms_zip(dsyms_zip.name, project)
                 logger.debug("Uploaded dSYMs for build %s", build)
+                metrics.incr("tasks.app_store_connect.builds_ingested", sample_rate=1)
```

Comment:
would it be useful to also track the number of skipped builds as well, for comparison against the actual number of builds successfully processed?

Reply:
What do you mean with skipped builds?  All other branches of this `else` result in sentry errors, those that don't i think are intentional so not really skipped so i didn't think i wanted a metric, but maybe you disagree?

Reply:
let's leave this as-is.